test-framework/junit5/src/main/java/io/quarkus/test/junit/NativeImageTest.java

```diff
@@ -20,6 +20,9 @@
  * run, it is expected that the JVM tests will be standard unit tests that are
  * executed by surefire, while the native image tests will be integration tests
  * executed by failsafe.
+ * This also means that injection of beans into a test class using {@code @Inject} is not supported
+ * in native image tests. Such injection is only possible in tests injected with
+ * {@link @QuarkusTest} so the test class structure must be take this into account
```

Comment:
```suggestion
 * {@link @QuarkusTest} so the test class structure must take this into account.
```

Reply:
It wouldn't have been a proper PR of mine unless it had a typo :). Thanks for spotting it